story/marginalia.md

```diff
@@ -0,0 +1,8 @@
+As I look through the rows upon rows of books lined up on the shelves, I peek pull a few to rifle through the pages. Some are very dusty, and haven't been touched in years. Others include fleeting notes in the margins. One particular book was quite worn. The edges of its pages were tattered, the corners were dog-eared, and numerous sections were marked with receipts and a train ticket. Behind the front cover there was a handwritten note:
+
+```
+Dearest Clara,
+I was not expecting to meet someone as exciting as you on this trip. I really enjoyed our time together, as brief as it may have been.
+
+<3 Fredo
+```
```

Comment:
Switch to quotes instead of preformatted text.
```suggestion
> Dearest Clara,
> I was not expecting to meet someone as exciting as you on this trip. I really enjoyed our time together, as brief as it may have been.
>
> <3 Fredo
```